cmd/query/main.go

```diff
@@ -64,22 +64,18 @@ func main() {
 			baseFactory := svc.MetricsFactory.Namespace(metrics.NSOptions{Name: "jaeger"})
 			metricsFactory := baseFactory.Namespace(metrics.NSOptions{Name: "query"})
 
-			tracer, closer, err := jaegerClientConfig.Configuration{
-				ServiceName: "jaeger-query",
-				Sampler: &jaegerClientConfig.SamplerConfig{
-					Type:  "probabilistic",
-					Param: 1.0,
-				},
-				RPCMetrics: true,
-			}.NewTracer(
+			cfg, err := jaegerClientConfig.FromEnv()
```

Comment:
Unfortunately, this is not backwards compatible. This really needs to be fixed in the client, so that FromEnv could be called on an already populated config as a for of override.

Reply:
Making changes to client for this single application seems like an overkill
 I've made changes in tracer config initialization, should be backwards-compatible now